assets/scss/pages/_alerts-page.scss

```diff
@@ -56,7 +56,7 @@
   justify-content: space-between;
   flex-basis: min-content;
   padding-bottom: 30px;
-  border-bottom: 1px solid govuk-colour('dark-grey');
+  border-bottom: 1px solid govuk-colour('black');
```

Comment:
This is the wrong rule - need to change the font colour for the dates. It's in the `&__metadata` rule
```
  &__metadata {
    font-size: 16px;
    font-weight: 300;
    color: $govuk-secondary-text-colour;
  }
```